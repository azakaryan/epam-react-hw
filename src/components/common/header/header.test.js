import React from 'react';
import { shallow } from 'enzyme';
import expectEnzymify from 'expect';
import enzymify from 'expect-enzyme';
import Header from './header';

expectEnzymify.extend(enzymify());

describe('Header Component', () => {
  it('should render correctly', () => {
    const component = shallow(<Header />);
    expect(component).toMatchSnapshot();
  });

  it('renders a Header element', () => {
    const wrapper = shallow(<Header />);
    expectEnzymify(wrapper).toBeA('div');
  });
});
