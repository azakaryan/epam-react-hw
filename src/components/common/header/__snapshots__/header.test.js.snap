// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component should render correctly 1`] = `
<div
  className="main"
/>
`;
